yb-voyager/cmd/importDataStatusCommand.go

```diff
@@ -318,3 +323,65 @@ func prepareRowWithDatafile(dataFile *datafile.FileEntry, state *ImportDataState
 	}
 	return row, nil
 }
+
+func getErrorHandlerUsed() (importdata.ImportDataErrorHandler, error) {
+	switch importerRole {
+	case IMPORT_FILE_ROLE:
+		return getImportDataFileErrorHandlerUsed()
+	case TARGET_DB_IMPORTER_ROLE:
+		return getImportDataErrorHandlerUsed()
+	default:
+		return nil, fmt.Errorf("unknown importer role: %s", importerRole)
+	}
+}
+
+func isErrorPolicyUsed() (bool, error) {
+	switch importerRole {
+	case IMPORT_FILE_ROLE:
+		idfsr, err := metaDB.GetImportDataFileStatusRecord()
+		if err != nil {
+			return false, fmt.Errorf("error while getting import data file status record: %w", err)
+		}
+		if idfsr == nil {
+			return false, nil
+		}
+		return idfsr.ErrorPolicy != "", nil
+	case TARGET_DB_IMPORTER_ROLE:
+		idsr, err := metaDB.GetImportDataStatusRecord()
+		if err != nil {
+			return false, fmt.Errorf("error while getting import data status record: %w", err)
+		}
+		if idsr == nil {
+			return false, nil
+		}
+		return idsr.ErrorPolicySnapshot != "", nil
+	default:
+		return false, fmt.Errorf("unknown importer role: %s", importerRole)
+	}
+}
+
+func getImportDataErrorHandlerUsed() (importdata.ImportDataErrorHandler, error) {
+	errorPolicyUsedStr, err := metaDB.GetImportDataErrorPolicySnapshotUsed()
+	if err != nil {
+		return nil, fmt.Errorf("error while getting import data error policy snapshot used: %w", err)
+	}
```

Comment:
Can we combine these two functions and do the check if `errorPolicyUsedStr != ""` here itself and return nil as the error handler, and just keep a check for nil error handler at the place we are calling this function? This will avoid any unnecessary errors. 
e.g., you are using the `getImportDataErrorHandlerUsed` in the get data migration report without checking if an error policy is provided, which might result in some error in some weird scenario where the error policy is not present.